.github/PULL_REQUEST_TEMPLATE/significant_changes.md

```diff
@@ -0,0 +1,25 @@
+<!--- Use this only if you've made significant changes to ACE's inner workings. -->
+
+## Description
+<!--- Describe your changes in few sentences. If your pull request is related to an issue, please put `Closes #issueNumber` or `Related to #issueNumber` sentence here. -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Especially point out in which productions it was used so far -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
```

Comment:
New feature should be done as new component or as significant change?

Reply:
significant change, unless component requires it and comes in same PR